index.html

```diff
@@ -38,8 +40,130 @@ <h2>I am a Software Developer</h2>
         </ul>
         </nav>
         <a class="bg_img"></a>
-  </footer>
-  </section>
+      </article>
+    </section>
+
+    <section class="works">
+      <section class="title">My Recent Works</section>
+      <hr>
+      <section class="feature">
+        <a class="img"></a>
+        <section class="describe">
+          <h2>Multi-Post Stories</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          <ul>
+            <li>CSS</li>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+      </section>
+
+      <section class="frame">
+        <section class="card card1">
+          <h1>Professional Art<br>Printing data</h1>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores ea laborum praesentium ad recusandae fugit deserunt explicabo quisquam voluptatum debitis nisi reprehenderit repellat, consequatur eaque esse quos earum molestiae? Ea?</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+        <section class="card card2">
+          <h1>Professional Art<br>Printing data</h1>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores ea laborum praesentium ad recusandae fugit deserunt explicabo quisquam voluptatum debitis nisi reprehenderit repellat, consequatur eaque esse quos earum molestiae? Ea?</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </section>
+        <section class="card card3">
+          <h1>Professional Art<br>Printing data</h1>
```

Comment:
- [ ] To help improve your page SEO, it is best practice to have only one h1 tag on an HTML page. Kindly change all the h1 tags in each project card to any other hx, could be h2.